mopidy_spotify/browse.py

```diff
@@ -22,15 +28,16 @@
 }
 
 _TOPLIST_REGIONS = {
-    "user": lambda session: spotify.ToplistRegion.USER,
     "country": lambda session: session.user_country,
     "everywhere": lambda session: spotify.ToplistRegion.EVERYWHERE,
 }
 
 
-def browse(config, session, uri):
+def browse(config, session, webclient, uri):
```

Comment:
s/webclient/web_client/ throughout the file

Also, it's starting to become a lot of arguments. Maybe at `*` at the beginning of the argument liste to force kwargs usage?